controllers/configmap_controller.go

```diff
@@ -22,9 +22,11 @@ import (
 	"fmt"
```

Comment:
To make this more reviewable it would be helpful to describe what you are attempting to do in the commit message.